src/utils/num.ts

```diff
@@ -206,27 +287,52 @@ export function getHexStringArray(value: Array<string>) {
  * // result2 = "0"
  * ```
  */
-export const toCairoBool = (value: boolean): string => (+value).toString();
+export function toCairoBool(value: boolean): string {
```

Comment:
Why replace arrow function ?

Reply:
Consistency, everything else was in export function format. If this one was that way because of the size of the func, there are other 1-liner funcs in the file that weren't arrow.